Dockerfile-legacy

```diff
@@ -96,7 +96,7 @@ RUN apt-get update -q && \
     pecl install yaml-1.3.0 && \
     echo "extension=yaml.so" > $CONF_PHPMODS/yaml.ini && \
     # Install new PHP5-stable version of Redis \
-    pecl install redis-2.2.8 && \
+    pecl install redis-3.1.0 && \
```

Comment:
I take it there are built-in tests for the 'legacy' php older than php 7 in here?